docs/oauth.md

```diff
@@ -199,60 +249,93 @@ This set of OAuth Config capabilities is not shared with the regular connector c
             <value>username</value>
         </entry>
     </accessTokenResponseMaps>
-    <!-- No refreshTokenResponseMaps needed since that's same with the accessTokenResponseMaps -->
+
+    <!-- No refreshTokenResponseMaps needed since it's the same as accessTokenResponseMaps -->
  </pluginOAuthConfig>
 
 ```
-# OAuth on Tableau Desktop
 
-In Tableau Desktop the connection dialog will look like this.  If a different auth mode is chosen, notice different required fields appear. This example has multiple auth modes supported and one of them is OAuth.
+## Instance URL/Custom Domain
+Some IDPs have a single global endpoint, such as https://accounts.google.com/o/oauth2/. Others have different instances, for example Okta would have a different instance URL for each customer. If your authorization server or IDP has different instances, then enable OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN. In this case the end users will be prompted for the URL when connecting.
```

Comment:
If your authorization server or IDP has different instances, then either enable OAUTH_CAP_SUPPORTS_CUSTOM_DOMAIN for embedded idp, or create multiple external/custom oauth configs.